cmd/avalanche.go

```diff
@@ -19,6 +19,7 @@ var (
 	metricInterval = kingpin.Flag("metric-interval", "Change __name__ label values every {interval} seconds.").Default("120").Int()
 	port           = kingpin.Flag("port", "Port to serve at").Default("9001").Int()
 	remoteURL      = kingpin.Flag("remote-url", "URL to send samples via remote_write API.").URL()
+	remoteTenant   = kingpin.Flag("remote-tenant", "Tenant ID to include in remote_write send").Default("1").String()
```

Comment:
Why 1 instead of 0? The default Cortex startup guide uses 0 as the tenant for all the data.

Reply:
I did not know that.  It's 1 here because I have some other tooling that treats 0 as an error.
I can change the default.

Reply:
:+1: Let's keep the default as 0 then this is good to merge. Thanks!